app/Http/Controllers/Widgets/SyslogController.php

```diff
@@ -43,6 +44,8 @@ public function getSettingsView(Request $request)
 
         $data['device'] = Device::hasAccess($request->user())->find($data['device']);
 
+        $data['priorities'] = array_flip(app('translator')->get('syslog.severity'));
```

Comment:
You don't need to array_flip() here, just flip $name and $val in syslog.blade.php foreach

Reply:
Done as requested.